tahowallet/extension
# NFT marketplace links revamp

Resolves https://github.com/tahowallet/extension/issues/3197
### What
- add Rarible marketplace
![image](https://user-images.githubusercontent.com/20949277/228249229-3ea091c1-7c2a-41b3-9470-de60ba33b763.png)
- use big icons instead of buttons for NFT preview page, add tooltip
![image](https://user-images.githubusercontent.com/20949277/228249363-ceb4a3b9-13ab-4704-8ccb-c92f476bf0e3.png)
- replace png icons with SVG

### Testing
- make sure marketplaces' links are looking good
- make sure correct marketplaces are displayed - rarible for Ethereum and Polygon NFTs, Galaxe for badges etc
